### ISSUE TITLE
DoTA FPS and annotations

### ISSUE THREAD

**Issue by orsveri** [monjurulkarim/risky_object#13]
Thank you for sharing such a nice method and providing all the data and clear instructions! 
I was able to reproduce the reported results using the best model you provided both on ROL and DoTA (after finetuning).
However, I have a question regarding FPS (frames per second) settings during training. 

If I understand correctly, ROL data has an FPS of 20 and the model was initially trained for this frequency. 
However, DoTA has an FPS of 10 by default, when you download pre-extracted frames. Did you use pre-extracted frames or raw videos? 

So, in short, what is the frequency of the DoTA preprocessed data (.npz) you provided? 
Also, how did you create DoTA gt labels (object-level "safe"/"dangerous" labels) that are used for fine-tuning?
